src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,43 @@
+.nav {
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+  &__item:not(:last-child) {
+      margin-right: 20px;
+    }
+  &__link {
```

Comment:
somethings is wrong with formatting, try to use prettier,
add empty lines between selectors